inc/Engine/Common/Ajax/AjaxHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace WP_Rocket\Engine\Common\Ajax;
+
+class AjaxHandler {
+
+	/**
+	 * Validate the referer before going futher.
+	 *
+	 * @param array $args Arguments to configure the validation.
+	 *
+	 * @return bool
+	 */
+	public function validate_referer( array $args ) {
+
+		if ( key_exists( 'referer', $args ) && is_string( $args['referer'] ) ) {
+			check_admin_referer( $args['referer'] );
```

Comment:
I think the index name should be `action` instead of `referer` to match with the parameter name of the function